### PR TITLE
IBX-23: Taxonomy

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-23
| Versions      | master, 4.0

### CODE REVIEW

File: docs/taxonomy.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Taxonomy
+
+Most content can be structured in logical ways, using categorization or labelling.
+Tags enable you to organize content according to different keywords (tags) and relationships between them, making it easier for your site users to find content they are interested in. They also allow you to create dynamic content pulls to display information customized for your pages and users.
+
+## Taxonomy tree
+
+In the site tree in the left pane, you can create a hierarchy of all tags and navigate in the platform.
```
Comment (webhdx) [inline docs/taxonomy.md:8]: `pane` typo?

File: docs/taxonomy.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+1. Add tag's identifier.
+1. (Optional) Add tag's description.
+1. To set the parent category, click the **Select parent** button.
+1. To save a new tag, click **Save**.
+
+Now, the tag is visible in the taxonomy tree.
+
+### Assign tag to content from taxonomy tree
+
+You can assign tags in three ways.
```
Comment (webhdx) [inline docs/taxonomy.md:27]: There is only one way to assign tags to Content - via content create/edit form.

File: docs/taxonomy.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Open Content Type, scroll down, in the section Select tags, you can add tag by typing or click **Select from tree**.
```
Comment (webhdx) [inline docs/taxonomy.md:29]: Open Content item? Or better edit/create content item?

**Review by webhdx — APPROVED** [review]

File: docs/creating_content_basic.md [hunk 4]
```diff
@@ -102,7 +102,20 @@ Here you can also add a link to the text. You can link to an external website, o
 While editing, you can preview what the content will look like by clicking the **Preview** button in the menu.
 
 A Content item can have different looks for different [SiteAccesses](creating_content_advanced.md#siteaccess).
-You can select a SiteAccess to preview by using a dropdown in the preview screen.
+You can select a SiteAccess to preview by using a drop-down in the preview screen.
+
+### Add taxonomy entries
+
+To keep your content organized and easy to find, you can add taxonomy entries to a content while creating or editing it.
```
Comment (DominikaK) [inline docs/creating_content_basic.md:109]:
```suggestion
To keep your content organized and easy to find, you can add taxonomy entries to a Content item while creating or editing it.
```

File: docs/taxonomy.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+You can add many tags.
+
+See [Add taxonomy entries](creating_content_basic.md#add-taxonomy-entries).
+
+### Add tag translation
+
+If you have more than one language translations of your site, you can translate also tags used in the site into the target language.
+
+1. Go to **Tags** section.
+1. Click the respective tag and go to the **Translations** tab.
```
Comment (DominikaK) [inline docs/taxonomy.md:37]:
```suggestion
1. Click a tag and go to the **Translations** tab.
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

**Review by DominikaK — APPROVED** [review]

**Pull request merged**